trieste/models/gpflux/optimizer.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import tensorflow as tf
+from gpflux.models import DeepGP
+from tensorflow.python.data.ops.iterator_ops import OwnedIterator as DatasetOwnedIterator
+
+from ..optimizer import LossClosure, TrainingData, create_loss_function
+
+
+@create_loss_function.register
```

Comment:
in this new version with `fit` method where is this used exactly?

Reply:
Forgot to remove this...